app/web/src/newhotness/StatusPanelIcon.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <Transition
+    enterActiveClass="transition-opacity duration-300"
+    enterFromClass="opacity-0"
+    leaveActiveClass="transition-opacity duration-500"
+    leaveToClass="opacity-0"
+    appear
+  >
+    <!-- We may want to make this "invisible" to keep the space in the future. -->
```

Comment:
Yeah, so... right now it just disappears. I think that's fine, but if we get a lot of element popping in the future, we may just want to reserve the space.